V2/tests/services/activity_service_test.py

```diff
@@ -0,0 +1,82 @@
+from V2.source.services.activity_service import ActivityService
+from V2.source.daos.activities_dao_interface import ActivitiesDaoInterface
+from V2.source.dtos.activity import ActivityDto
+from http import HTTPStatus
+
```

Comment:
Creo que este módulo debería tener solo dos tests, uno para get_all y otro para get_by_id. Te escribo para get_all pero te dejo a ti el de get_by_id:

```suggestion
def test__get_all__uses_the_dao_to_retrieve_activities(mocker):
    expected_activities = mocker.Mock()
    dao = mocker.Mock(get_all=mocker.Mock(return_value=expected_activities))
    service = ActivityService(dao)
    
    acutal_activities = service.get_all()
    
    assert dao.get_all.called
    assert expected_activities == actual_activities
```

Reply:
Done!